src/Ledger/Ledger.lagda

```diff
@@ -73,47 +81,55 @@ allColdCreds govSt es =
 \caption{Types and functions for the LEDGER transition system}
 \end{figure*}
 \begin{code}[hide]
-instance
-  unquoteDecl To-LEnv To-LState = derive-To
-    ((quote LEnv , To-LEnv) ∷ (quote LState , To-LState) ∷ [])
-
 private variable
   Γ : LEnv
   s s' s'' : LState
-  utxoSt' : UTxOState
-  govSt' : GovState
-  certState' : CertState
+  utxoSt utxoSt' : UTxOState
+  govSt govSt' : GovState
+  certState certState' : CertState
   tx : Tx
+
 \end{code}
 
 \begin{figure*}[ht]
-\begin{AgdaSuppressSpace}
+\begin{AgdaMultiCode}
 \begin{code}
 data _⊢_⇀⦇_,LEDGER⦈_ : LEnv → LState → Tx → LState → Type where
 
   LEDGER-V :
-    let 
-      open LState s
-      txb = tx .body
-      open TxBody txb
-      open LEnv Γ
-      open CertState certState
-      open DState dState
+    let  txb         = tx .body
+\end{code}
+\begin{code}[hide]
+         open TxBody txb
+\end{code}
+\begin{code}
+         pp          = Γ .pparams
+         slot        = Γ .slot
+         ppolicy     = Γ .ppolicy
+         enactState  = Γ .enactState
```

Comment:
I haven't tried it, but I feel this would look better as a fake pattern-match below, now that `Γ` is only used via its fields.